python_modules/dagster/dagster/_core/execution/plan/active.py

```diff
@@ -249,6 +267,32 @@ def _update(self) -> None:
             self._executable.append(key)
             del self._waiting_to_retry[key]
 
+    def is_provenance_changed(self, step: ExecutionStep) -> bool:
+        existing_combined_input_versions: Dict[AssetKey, LogicalVersion] = {}
```

Comment:
It's worth adding some docstring here to explain what this function is intended to do. Is it something like "determine whether versions of any of the inputs of the assets targeted by the step have changed"?